### DISCUSSION TITLE
pubg MK TOOL  Ipad view problem.

### DISCUSSION THREAD

**Discussion by ajrz007** [MohamedKVIP/MK-PUBG-Mobile-Tool#14] [General]
sir thanks because of making this tool. i'm playing pubg gameloop in monitor UHD 3440x1440 and i'm not able to active ipad view via this app. please add 3440 option or tell me another way. thank you so much .

with regards
ajrz

**Comment by MohamedKVIP** [comment]
Not an issue. It's resolved in the AIO version (Discord), which has custom resolution support.

**Reply by ajrz007** [reply]
how to access AIO version? and how to find your Discord?

**Reply by MohamedKVIP** [reply]
This is Discord Server [https://discord.gg/PDPJM6e6PC](https://discord.gg/PDPJM6e6PC)

**Comment by ajrz007** [comment]
Sir i bought aio verison
But still i'm not able to use ipadview in my ultrawide monitor (3440×1440)
@MohamedKVIP
